c2corg_api/views/__init__.py

```diff
@@ -126,7 +128,13 @@ def to_json_dict(obj, schema, with_special_locales_attrs=False):
             for attr in locale_special_attributes:
                 if hasattr(locale, attr):
                     locale_dict[attr] = getattr(locale, attr)
-
+    if (with_special_geometry_attrs and obj.type in(ROUTE_TYPE, OUTING_TYPE)):
```

Comment:
Do you really need the parenthesis for the condition? A blank space after ``in`` is recommended I think.
Perhaps you could write:
```
if with_special_geometry_attrs and obj.type in (ROUTE_TYPE, OUTING_TYPE):
```

Reply:
Not need, I added it before because it was a multiline condition, I will remove it